docs/use-case1.rst

```diff
@@ -1,69 +1,80 @@
-type of :code:`integration-service`-mediated bridge, by putting into communication a :code:`ROS2` *talker-listener*
-example with a :code:`Fast-RTPS` *HelloWorld* example.
+This section is intented to illustrate the *DDS*-*ROS2* communication as an example of this
+type of *eProsima Integration-Service*-mediated bridge, by putting into communication a *ROS2* *talker-listener*
+example with a *Fast-RTPS* HelloWorld example.
 
 .. image:: DDS_WITH_IS.png
 
 Example: ROS2 communication
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 To prepare the deployment and setup the environment correctly, please follow the introductory steps delined in
-:ref:`Getting Started <getting started>` and read carefully the :ref:`Important reminders <important reminders>`
+:ref:`Getting Started <getting started>` and read carefully the :ref:`Important remarks <important remarks>`
 section.
 
 Also, to get this example working, the following additional requirements must be met:
 
-- Having :code:`ROS2` (Crystal or superior) installed, with the *talker-listener* example working.
-- Having :code:`Fast-RTPS` installed (at least v1.9.2), with the *HelloWorld* example working.
-
-**Note**: If you built the :code:`integration-service` and/or :code:`ROS2` packages with colcon, please make sure
-to have done all the required sourcing of the colcon overlays or, in alternative, to have added the opportune
-source commands to the .bashrc file, as explained in the :ref:`Getting Started <getting started>` section.
+- Having *ROS2* (Crystal or superior) installed, with the *talker-listener* example working.
+- Having *Fast-RTPS* installed (at least v1.9.2), with the HelloWorld example working.
 
 Open three terminals:
 
-- In the first terminal, execute a :code:`ROS2` *talker*
+- In the first terminal, execute a *ROS2* :code:`talker`
 
 .. code-block:: bash
 
     ros2 run demo_nodes_cpp talker
 
-- In the second terminal, execute the :code:`Fast-RTPS` HelloWorld *subscriber*
+- In the second terminal, execute the *Fast-RTPS* HelloWorld :code:`subscriber`
 
 .. code-block:: bash
 
     ./HelloWorldExample subscriber
 
 At this point, the two applications cannot communicate due to the incompatibility of
-their **topic** and **type** in their :code:`DDS` configuration. This is where :code:`integration-service` comes
-into play to make this communication possible.
+their **topic** and **type** in their *DDS* configuration. This is where *eProsima Integration-Service* comes
+into play to make the communication possible.
 
-- In the third terminal, execute :code:`integration-service` using the :code:`dds_ros2_string.yaml` configuration file
-  located in the :code:`soss-dds/examples/udp/` folder.
+- In the third terminal, execute *eProsima Integration-Service* using the :code:`soss` command and with the
+  :code:`dds_ros2_string.yaml` configuration file located in the :code:`soss-dds/examples/udp/` folder.
```

Comment:
Here, I'm missing an easy way to give a glance at the YAML file, maybe in an inline code comment or (better IMO) using a link to the github repository containing the file: https://github.com/eProsima/SOSS-DDS/blob/feature/xtypes-dds/examples/udp/dds_ros2_string.yaml

Reply:
ok.

Reply:
gasco.

Reply:
Most of these YAML files don't exist yet. I'll add a TODO comment in all cases, to link them once they are created.